newIDE/app/src/Credits/CreditsPackagesDialog.js

```diff
@@ -0,0 +1,231 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import * as React from 'react';
+import { Column } from '../UI/Grid';
+import GDevelopThemeContext from '../UI/Theme/GDevelopThemeContext';
+import {
+  ColumnStackLayout,
+  LineStackLayout,
+  ResponsiveLineStackLayout,
+} from '../UI/Layout';
+import FlatButton from '../UI/FlatButton';
+import Text from '../UI/Text';
+import Dialog from '../UI/Dialog';
+import CreditsStatusBanner from './CreditsStatusBanner';
+import { CreditsPackageStoreContext } from '../AssetStore/CreditsPackages/CreditsPackageStoreContext';
+import PlaceholderError from '../UI/PlaceholderError';
+import PlaceholderLoader from '../UI/PlaceholderLoader';
+import RaisedButton from '../UI/RaisedButton';
+import { formatProductPrice } from '../AssetStore/ProductPriceTag';
+import BackgroundText from '../UI/BackgroundText';
+import Link from '../UI/Link';
+import Window from '../Utils/Window';
+import CreditsPackagePurchaseDialog from '../AssetStore/CreditsPackages/CreditsPackagePurchaseDialog';
+import { type CreditsPackageListingData } from '../Utils/GDevelopServices/Shop';
+import Coin from './Icons/Coin';
+import TwoCoins from './Icons/TwoCoins';
+import ThreeCoins from './Icons/ThreeCoins';
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
+import AlertMessage from '../UI/AlertMessage';
+
+const styles = {
+  creditsPackage: {
+    display: 'flex',
+    flex: 1,
+    borderRadius: 8,
+    padding: 16,
+  },
+  titleContainer: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  iconContainer: {
+    marginRight: 8,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: 30,
+  },
+  backgroundText: {
+    textAlign: 'left',
+  },
+};
+
+const getIconFromIndex = (index: number) => {
```

Comment:
Nit: Should that be something like `getIconFrom[Something that could be changed backend side]`?

Reply:
I thought about that, but wasn't sure it was worth making the icons changed from the backend side, so prefered keeping a design "1/2/3/3.." for coins